src/formElementPlugins/pluginProvider.tsx

```diff
@@ -45,6 +45,13 @@ class pluginProvider {
         folderName: 'textInfo',
         category: 'Informative',
       },
+      imageDisplay: {
+        code: 'imageDisplay',
+        isCore: true,
+        displayName: 'Image Display',
```

Comment:
Could be called just "Image"

Reply:
I dunno, it kind of seems like a parallel name to "Text Info". Just "Image" as a name is a bit too ambiguous, don't you think? Could be image upload, etc.